mal/cli.py

```diff
@@ -134,6 +134,25 @@ def create_parser():
                             help='add anime with this status (e.g "on hold")')
     parser_add.set_defaults(func=commands.add)
 
+    # Parser for "edit" command
+    parser_edit = subparsers.add_parser('edit',
+                                        help='edit entry')
+    parser_edit.add_argument('anime_regex',
+                             help='regex pattern to match anime titles')
+    parser_edit.add_argument('--score', type=int)
+    parser_edit.add_argument('--status',
+                             help='status to assign to entry',
+                             choices=['watching', 'completed',
+                                      'on hold', 'dropped',
+                                      'plan to watch', 'rewatching'])
+    parser_edit_tags = parser_edit.add_mutually_exclusive_group()
+    parser_edit_tags.add_argument('--tags', nargs='+',
```

Comment:
Since we have `--add-tags` command I prefer that this be `--set-tags` to be more explicit what does.

Reply:
Good point